### PR TITLE
Instructions for using snapcraft on Ubuntu 14.04.

### PR DESCRIPTION
**Pull request by evandandrea**
I put together some instructions for using snapcraft on Ubuntu 14.04. I've tested both these journeys, but additional attempts wouldn't hurt.

Note that this only offers a way to build the snap. I contemplated including instructions for using Thomas Voss' snapd-on-trusty PPA, but decided to wait for that to land in 14.04 proper.

### CODE REVIEW

**Comment by evandandrea** [comment]
@elopio has just clued me in to instructions for [running snapd on LXD](https://www.stgraber.org/2016/12/07/running-snaps-in-lxd-containers/). I'll update this PR with them worked in.

File: build-snaps/trusty.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Building and testing snaps on Ubuntu 14.04"
+---
+
+
+In the near future snapcraft will be offered as a snap, giving you the same experience regardless of Linux distribution or release. Until then, you can build snaps on Ubuntu 14.04 using LXD or Docker.
+
+If you're already running Ubuntu 16.04 or later, you can skip this guide and run snapcraft directly.
```
Comment (anthonydillon) [inline build-snaps/trusty.md]: Can you add a link to `run snapcraft directly`. Back to the original guide please.

**Comment by evandandrea** [comment]
More on running snapd on LXD:

> 4:35 PM \<stgraber> should work fine on 16.10, will work fine on 16.04 soon-ish (next couple of kernel SRUs I'm told)

File: build-snaps/trusty.md (hunk 1 above)
Comment (evandandrea) [inline build-snaps/trusty.md]: Done, thanks for the suggestion.

**Pull request merged**